metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1486,13 +1487,27 @@ def extract_independent_looks(self) -> Iterable[MetadataWorkUnit]:
         )
         for look in all_looks:
             if look.id in self.reachable_look_registry:
-                # This look is reachable from Dashboard
+                # This look is reachable from the Dashboard
                 continue
 
             if look.query_id is None:
                 logger.info(f"query_id is None for look {look.title}({look.id})")
                 continue
 
+            if self.source_config.skip_personal_folders:
+                if look.folder is not None and (
+                    look.folder.is_personal or look.folder.is_personal_descendant
+                ):
+                    self.reporter.info(
```

Comment:
What is the benefit of having this reporter info vs collecting these in the report below and add a debug log?

Reply:
This is to maintain consistency with the logic used in the personal dashboards.